app/models/tax_form_due_date.rb

```diff
@@ -1,2 +1,14 @@
 class TaxFormDueDate < ApplicationRecord
+
+  validates_uniqueness_of   :name
```

Comment:
wait, this is a problem because we validate uniqueness in three different columns : name, state, and year. not just name

Reply:
As discussed. Noted for future